var/spack/repos/builtin/packages/zoltan/package.py

```diff
@@ -153,6 +153,12 @@ def configure_args(self):
                     f"--with-scotch-libdir={scotch_prefix.lib}",
                 ]
             )
+            for a in (config_libs, config_ldflags):
+                a.append("-lptscotch")
+                a.append("-lptscotcherr")
+                a.append("-lscotch")
+                a.append("-lscotcherr")
+                a.append("-lscotcherrexit")
```

Comment:
It might be better to just do:
```
config_libs.append(spec["scotch"].libs.ld_flags)
config_ldflags.append(spec["scotch"].libs.ld_flags)
```

Reply:
Thanks. That works and is much better. I'll push a change.